tests/ut-ncnnapi.cc

```diff
@@ -165,7 +166,7 @@ TEST(ncnnapi, service_lstm)
   mkdir(csvts_repo.c_str(), 0777);
   std::string sname = "my_service_csvts";
   std::string jstr
-      = "{\"mllib\":\"caffe\",\"description\":\"my ts "
+      = "{\"mllib\":\"ncnn\",\"description\":\"my ts "
```

Comment:
the idea here was to learn and predict with caffe, then to compare to ncnn prediction, here you try to learn with ncnn ?